spec/features/guests/show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe "Guest Show Page" do
+    it "can display the guest and the list of rooms they've stayed in" do
+        hotel = Hotel.create!(name: "Caesar's")
+        room1 = hotel.rooms.create!(suite: "Presidential", rate: 400)
+        room2 = hotel.rooms.create!(suite: "Deluxe", rate: 200)
+        guest = Guest.create!(name: "John Lennon")
+        Booking.create!(guest: guest, room: room1)
+        Booking.create!(guest: guest, room: room2)
+
+        visit "/guests/#{guest.id}"
+
+        expect(page).to have_content("John Lennon")
+        expect(page).to have_content("Presidential")
+        expect(page).to have_content("400")
+        expect(page).to have_content("Deluxe")
+        expect(page).to have_content("200")
+        expect(page).to have_content("Caesar's")
```

Comment:
This works, however as we continue to move on, I'd like to see your tests get more specific. So instead of just testing for the number "400" on a page, which could be anywhere, we'll want to test for labels and other content that comes with the number, like "Room Rate: 400", or use within blocks to zero in on where this content should be located.